cmd/keymasterd/idp_oidc_test.go

```diff
@@ -216,32 +216,43 @@ func TestIdpOpenIDCClientCanRedirectFilters(t *testing.T) {
 		AllowedRedirectDomains: []string{"example.com"},
 	}
 	state.Config.OpenIDConnectIDP.Client = append(state.Config.OpenIDConnectIDP.Client, weakREWithDomains)
+	onlyDomainConfig := OpenIDConnectClientConfig{
+		ClientID:               "onlyWithDomains",
+		AllowedRedirectURLRE:   []string{"https://[^/]*\\.example\\.com"},
```

Comment:
It seems that `AllowedRedirectURLRE` should *not* be specified for this one?

Reply:
yes... fixed